apps/web/src/stores/evaluationResultCharts/evaluationResultsCounters.ts

```diff
@@ -12,23 +14,24 @@ export default function useEvaluationResultsCounters(
     documentUuid: string
     evaluationId: number
   },
-  opts: SWRConfiguration = {},
+  { fallbackData }: SWRConfiguration = {},
 ) {
   const { project } = useCurrentProject()
+  const fetcher = useCallback(async () => {
+    const [data, error] = await computeEvaluationResultsCountersAction({
```

Comment:
it's a pita so don't do it, but next time remember not to use server actions for fetching instead use regular api endpoints

Reply:
Why? I feel like having to use regular api endpoints are more pita than server actions.

Reply:
server actions are all post requests which doesn't allow for http caching, even react docs recommend not using server actions for fetching https://react.dev/reference/rsc/use-server#caveats

Reply:
I agree with pain in my heart